addons/libs/packets/fields.lua

```diff
@@ -2132,12 +2132,13 @@ fields.incoming[0x03C] = L{
     {ref=types.shop_item,       label='Item',               count='*'},         -- 08 -   *
 }
 
--- Price response
--- Sent after an outgoing price request for an NPC vendor (0x085)
+-- Price/sale response
+-- Sent in response to an outgoing price request for an NPC vendor (0x085), and in response to player finalizing a sale.
+-- Inventory bag is assumed to be regular Inventory (0)
 fields.incoming[0x03D] = L{
     {ctype='unsigned int',      label='Price',              fn=gil},            -- 04
     {ctype='unsigned char',     label='Inventory Index',    fn=invp+{0x09}},    -- 08
-    {ctype='unsigned char',     label='Bag',                fn=bag},            -- 09
+    {ctype='unsigned char',     label='Type'},                                  -- 09 0 = on apprasial, 1 = when sale is finalized
```

Comment:
Can you change `apprasial` to `price check` or something along those lines. 
There is an appraisal system in-game so using that word here is may lead to confusion.